web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -114,6 +114,10 @@ export const createAppConfigSchema = (platform: Platform) => {
       .boolean()
       .default(false)
       .describe('Enables sharing the computer.'),
+    'feature.headlessSkipConfirm': z
+      .boolean()
+      .default(false)
+      .describe('Skips the confirmation tab for headless login approval.'),
```

Comment:
```suggestion
      .describe('Skips the confirmation modal for headless login approval and asks for hardware key tap to confirm.'),
```

Something like this would convey better the nature of this config option.